torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -1179,6 +1175,36 @@ def call_triton_kernel(
                 "Please use a Config in @triton.autotune instead."
             )
 
+        # Run prune_configs_by to filter the configs
+        if isinstance(variable.kernel, Autotuner):
+            # args and kwargs aren't real values yet
+            # e.g., LazyVariableTracker
+            # If a user wants to access them, we should realize them
+            def realize_var(
+                var: VariableTracker,
+            ) -> Union[Any, VariableTracker, LazyVariableTracker]:
+                while isinstance(var, LazyVariableTracker) and hasattr(var, "realize"):
+                    var = var.realize()
+                # if we can return the value, do it
+                if hasattr(var, "value"):
+                    return var.value
+                elif hasattr(var, "get_real_value"):
+                    return var.get_real_value()
+                else:
+                    return var
+
+            realizedKwargs = {key: realize_var(kwarg) for key, kwarg in kwargs.items()}
+            realizedArgs = zip(
+                variable.kernel.arg_names, [realize_var(arg) for arg in args]
+            )
+            variable.kernel.nargs = dict(realizedArgs)
```

Comment:
as far as I can tell in Triton OSS this gets reset on each call to warmup(...) and run(...), used for a few funcs (prehook--unsupported, prune_configs, and warmup--unsupported)

I don't see any harm in saving the previous value though in case this gets changed